src/Tests/dotnet.Tests/dotnet-msbuild/GivenDotnetBuildInvocation.cs

```diff
@@ -54,8 +54,10 @@ public void MsbuildInvocationIsCorrect(string[] args, string expectedAdditionalA
 
         [Theory]
         [InlineData(new string[] { "-f", "tfm" }, "-target:Restore", "-property:TargetFramework=tfm")]
-        [InlineData(new string[] { "-p:TargetFramework=tfm" }, "-target:Restore", "-property:TargetFramework=tfm")]
+        [InlineData(new string[] { "-p:TargetFramework=tfm" }, "-target:Restore", "-p:TargetFramework=tfm")]
+        [InlineData(new string[] { "/p:TargetFramework=tfm" }, "-target:Restore", "-property:TargetFramework=tfm")]
```

Comment:
What's the reason that `/p` gets translated to `-property` but `-p` stays as is?

Reply:
`/p` had to be hardcoded as a option to work around a system command line issue, but `-p` works fine by just being parsed/forwarded as is https://github.com/dotnet/sdk/blob/master/src/Cli/dotnet/CommonOptions.cs#L15